### ISSUE TITLE
Request: Turn to magisk module for android

### ISSUE THREAD

**Issue by GoodyOG** [abdoxfox/HTTP-CUSTOM-HEADERS-VPN#35]
I'll want to request this to be like a magisk module, something like Box for android . https://github.com/taamarin/box_for_magisk. To start and stop, enable/disable the module in magisk manager. Thanks.

**Comment by John01-DEV** [comment]
Bro it's working? I tried here but doesn't work

**Comment by GoodyOG** [comment]
Yes it's working. Join their telegram and post any error u getting.
https://t.me/phreakingboss

**Comment by abhishekcs0** [comment]
Hello do you know how to use this on linux desktop, I am using debian, I am not able to find any working soulution for this, On android HTTP injector works flawless, please help
